src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+  private final int value;
+  public LottoNumber(int value) {
+    this.value = value;
+  }
+
+  @Override
+  public int compareTo(LottoNumber lottoNumber) {
+    if (this.value > lottoNumber.value) {
+      return 1;
+    }
+
+    if (this.value < lottoNumber.value) {
+      return -1;
+    }
+
+    return 0;
```

Comment:
중요한 부분은 아니지만 다음과 같이 간단하게 표현도 가능할 것 같아요!!😆
```suggestion
    return Integer.compare(value, lottoNumber.value)
```